video/out/win32/menu.c

```diff
@@ -178,9 +180,10 @@ static void build_menu(void *talloc_ctx, HMENU hmenu, struct mpv_node *node)
     }
 }
 
-struct menu_ctx *mp_win32_menu_init(void)
+struct menu_ctx *mp_win32_menu_init(HWND hwnd)
 {
     struct menu_ctx *ctx = talloc_ptrtype(NULL, ctx);
+    ctx->hwnd = hwnd;
     ctx->menu = CreatePopupMenu();
     ctx->ta_data = talloc_new(ctx);
     return ctx;
```

Comment:
ctx->update is not initialized.

Reply:
Also, `set menu-data null` should probably remove the already added submenu in system menu.